x/mint/spec/04_params.md

```diff
@@ -6,23 +6,28 @@ order: 4
 
 The `mint` module contains the following parameters:
 
-| Key                  | Type                | Example |
-|----------------------|---------------------|---------|
-| mint_denom           | string              | "stake" |
-| block_time_threshold | time.duration       | "10s"   |
-| inflation_schedules  | []InflationSchedule |         |
+| Key                  | Type                | Example                                         |
+|----------------------|---------------------|-------------------------------------------------|
+| mint_denom           | string              | "stake"                                         |
+| mint_pool_address    | string              | "cosmos17xpfvakm2amg962yls6f84z3kell8c5lserqta" |
+| block_time_threshold | time.duration       | "10s"                                           |
+| inflation_schedules  | []InflationSchedule |                                                 |
 
 ## MintDenom
 
 MintDenom is the denomination of the coin to be minted.
 
+## MintPoolAddress
+
+MintPoolAddress is the account address where the inflation generated by the mint module will accumulate. The default is FeeCollector, but if it set FeeCollector, it could mix the minted inflation with the collected tx fee. Therefore, it is recommended to specify a separate address depending on the chain and staking reward can be sent to the FeeCollector through Budget module functionality
```

Comment:
```suggestion
MintPoolAddress is the account address where the inflation generated by the mint module will accumulate. The default is FeeCollector, but if it set to FeeCollector, minted inflation could be mixed together with collected tx fees. Therefore, it is recommended to specify a separate address depending on the chain and staking reward can be sent to the FeeCollector through Budget module functionality
```

// mint_pool_address defines the address where inflation will be minted. The default is FeeCollector,
  // but if it is set to FeeCollector, minted inflation could be mixed together with collected tx fees.